virttest/utils_misc.py

```diff
@@ -3100,11 +3100,19 @@ def get_image_info(image_file):
                            'dsize' : '931135488',
                            'csize' : '65536'}
     """
+    def check_qemu_image_lock_support():
+        cmd = "qemu-img"
+        try:
+            binary_path = process.system_output("which %s" % cmd)
```

Comment:
You should use avocado.utils.path.find_command, which does the same and does not require calling a subprocess.

Reply:
Or virttest.utils_misc.find_command, which amounts to the same.